shared/slotutil/countdown.go

```diff
@@ -41,14 +41,14 @@ func CountdownToGenesis(ctx context.Context, genesisTime time.Time, genesisValid
 			ticker = time.NewTicker(time.Second)
 			secondTimerActivated = true
 		}
+		if timeRemaining >= time.Second {
+			log.WithFields(logFields).Infof(
+				"%s until chain genesis",
+				timeRemaining.Truncate(time.Second),
+			)
+		}
 		select {
 		case <-ticker.C:
-			if timeRemaining >= time.Second {
-				log.WithFields(logFields).Infof(
-					"%s until chain genesis",
-					timeRemaining.Truncate(time.Second),
-				)
-			}
```

Comment:
Wait, the ticker is supposed be the thing triggering the log, no?

Reply:
Now the ticker will just play a role of `time.Sleep`